### DISCUSSION TITLE
How to update or create a new permalink for a blog post?

### DISCUSSION THREAD

**Discussion by JamiePW** [academicpages/academicpages.github.io#2552] [Q&A]
I added a new .md blog file under /_posts, but it didn't show on the page. It seems that it doesn't have a permalink like the examples. What's the right workflow?

**Comment by rjzupkoii** [comment]
Once you push the updates to your repository and the site is deployed, you should be able to navigate to the page and the permalink will be in the navigation bar of your web browser. On the example site, this is one of the permalinks: https://academicpages.github.io/posts/2012/08/blog-post-4/

**Comment by JamiePW** [comment]
Now I have encountered another problem. The entire content of the .md file shows on the navigation page of blog posts. How should I "hide" the rest of the blog, leaving only abstract on the navigation page?

**Reply by rjzupkoii** [reply]
Can your clarify what you mean (or give a link to your site)? There are two types of pages that are generated for the blog posts - one has a short summary of all of them and a link to each of the individual posts, whereas the one is a single page for each post and has the permalink I mentioned before.
